####################
# ocr_info
####################

# Common header
Header header

# array of ocr result
ocr[] ocrs

# timestamp (in seconds)
float64 timestamp

# processing time (in seconds)
float64 processingtime
